data-prepper-plugin-framework/src/main/java/org/opensearch/dataprepper/plugin/ExtensionPluginConfigurationResolver.java

```diff
@@ -6,6 +6,7 @@
 package org.opensearch.dataprepper.plugin;
 
 import org.opensearch.dataprepper.model.configuration.PipelinesDataFlowModel;
+import org.springframework.beans.factory.annotation.Qualifier;
```

Comment:
We don't need this change and I'm guessing it will cause a build failure due to unused imports.